src/main/kotlin/org/wycliffeassociates/otter/common/persistence/repositories/WorkbookRepository.kt

```diff
@@ -63,7 +63,7 @@ class WorkbookRepository(private val db: IDatabaseAccessors) : IWorkbookReposito
         return db.getCollectionMetaContent(chapterCollection)
             .map { metaContent ->
                 Chapter(
-                    title = chapterCollection.titleKey,
+                    title = "Chapter " + chapterCollection.titleKey,
```

Comment:
Not sure if this is how we want to add "Chapter" or "Chunk" to the title (see below as well.) But, if we call this a title, I would prefer for it to already have "Chapter" or "Chunk" in the title, rather than just the chapter or chunk number.

Reply:
UPDATE: I reverted this commit since it was breaking the unit tests, and it may not be what we want to do anyway. I am now adding "Chapter" or "Chunk" to the titles in a domain function when we construct the `ResourceGroupCardItem`.